editor/src/components/PropertiesPanel/PanelDynamic/CheckboxProp.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <v-checkbox
+    small
+    dense
+    filled
+    no-gutters
+    class="caption"
+    :value="value"
+    @input="input"
+    @change="change"
+  >
+    <template v-slot:label>
+      <div class="caption">
+        {{ data.name }}
+      </div>
+    </template>
+  </v-checkbox>
+</template>
+
+<script lang="ts">
+import { defineComponent, ref } from '@vue/composition-api';
+import _ from 'lodash';
+const CheckboxProp = defineComponent({
+  name: 'CheckboxProp',
+  emits: ['input', 'change'],
+  props: ['value', 'config'],
+  setup(props: any, context: any) {
+    let data = ref(props.config);
```

Comment:
here maybe we can create a method in order to generate this of properties because  checkboxProp.vue, Dropdown.vue. Textprop, TextArea has this common code fragment.

please review if we can use composition in order to avoid the repeated code.